### ISSUE TITLE
Running with this renderer causes yath to blow up

### ISSUE THREAD

**Issue by autarch** [cpanel/Test2-Harness-Renderer-JUnit#3]
``` yath -r JUnit t/

** Defaulting to the 'test' command **

No time stamp found for event?!?!?!? at /home/autarch/.perlbrew/libs/perl-5.26.2@dev/lib/perl5/Test2/Harness/Renderer/JUnit.pm line 52, <$fh> line 1.
Killing runner

================================================================================

Run ID: 0F427840-F2A4-11E9-9E4F-97A54F7E7065
Can't use an undefined value as an ARRAY reference at /home/autarch/.perlbrew/libs/perl-5.26.2@dev/lib/perl5/App/Yath/Command/test.pm line 520, <$fh> line 1.
```

I tried this with a few different distros of mine, so I don't think it has anything to do with the actual tests being run.

I have Test2::Harness 0.001099 installed, along with the latest release of this distro.

**Comment by toddr** [comment]
The issue is a specific test2 event that I didn't have an example of so it was hard for me to document.

**Comment by toddr** [comment]
Can you give me anything I can run on my own to re-produce it?

**Comment by autarch** [comment]
```
> git clone git@github.com:CpanelInc/Test2-Harness-Renderer-JUnit.git
Cloning into 'Test2-Harness-Renderer-JUnit'...
remote: Enumerating objects: 61, done.
remote: Counting objects: 100% (61/61), done.
remote: Compressing objects: 100% (29/29), done.
remote: Total 61 (delta 13), reused 56 (delta 10), pack-reused 0
Receiving objects: 100% (61/61), 16.93 KiB | 5.64 MiB/s, done.
Resolving deltas: 100% (13/13), done.
 perl-5.26.2  autarch  houseabsolute  ~  projects  $  
> cd Test2-Harness-Renderer-JUnit/
 perl-5.26.2  autarch  houseabsolute  ~  projects  Test2-Harness-Renderer-JUnit  master  $  
> yath -r JUnit t

** Defaulting to the 'test' command **

No time stamp found for event?!?!?!? at /home/autarch/projects/Test2-Harness-Renderer-JUnit/lib/Test2/Harness/Renderer/JUnit.pm line 52, <$fh> line 1.
Killing runner

================================================================================

Run ID: BEB54778-F2A7-11E9-B3CE-27E94F7E7065
Can't use an undefined value as an ARRAY reference at /home/autarch/.perlbrew/libs/perl-5.26.2@dev/lib/perl5/App/Yath/Command/test.pm line 520, <$fh> line 1.
```

**Comment by toddr** [comment]
This is probably the same thing as #1

**Comment by toddr** [comment]
TLDR; each test event is supposed to have a time stamp (@exodist promised me!) That was a safety in the event one didn't.

**Comment by toddr** [comment]
This bisects to https://github.com/Test-More/Test2-Harness/commit/25259186f2af862ea14e8f71438c40d19daba0f2

which accidentally removed creation of timestamps in Test2/Harness/Feeder/Run.pm. This mostly affected event types `harness_job_launch` and `harness_job_end` which are critical for this module. I'm talking with @exodist now about a fix. in Harness.

**Comment by toddr** [comment]
This will be fixed by #7.
